spec/methods_spec.rb

```diff
@@ -424,7 +424,7 @@ def self.one; end
     expect(out).to eq("[\n    [0] none() Hello\n    [1]  one() Hello\n]")
   end
 
-  # See https://github.com/awesome-print/awesomer_print/issues/30 for details.
+  # See https://github.com/awesome-print/awesome_print/issues/30 for details.
```

Comment:
is it good to refer to the old project here?

Reply:
I think it's okay, since it's just a link for further information, and the link would not work with awesomer_print (new repository, new issue numbers).

Reply:
Yes, I understand that this repo doesn't have 30th issue for now :) 
But what if `awesome_print` will disappear later?
I think it's best to copy some text from the original issue to the source code.
Anyway, both solutions will work

Reply:
I think it's more likely to be archived than deleted.